### PR TITLE
Update unit tests for TimerDescriptionFieldHandler

### PR DESCRIPTION
**Pull request by AndrewVebster**
### 📒 Description
Updated and fixed unit tests related to the description shortcuts.

### CODE REVIEW

File: src/ui/osx/TogglDesktop/TogglDesktopTests/TimerDescriptionFieldHandlerTests.swift [hunk 1]
```diff
@@ -200,6 +200,22 @@ class TimerDescriptionFieldHandlerTests: XCTestCase {
         XCTAssertEqual(handler.state, State.autocompleteFilter("some @proj"))
     }
 
+    func testNotRecalculatingStateOnSameCursorPosition() {
```
Comment (skel35) [inline src/ui/osx/TogglDesktop/TogglDesktopTests/TimerDescriptionFieldHandlerTests.swift:203]: I'm not sure I understand the value of this test.
The way I see it is:
Scenario: text field string value is changing, although the cursor position doesn't change.
Expected: The autocomplete is not recalculated.

Is there a user scenario where this is supposed to happen?
To me, it looks like it's testing an implementation detail, but maybe I'm missing something.

Comment (AndrewVebster) [reply src/ui/osx/TogglDesktop/TogglDesktopTests/TimerDescriptionFieldHandlerTests.swift:203]: There is one use case:
The user removes all text -> Autocomplete shows all items -> Esc to hide autocomplete -> Left/right-arrow key -> 
Autocomplete will show on every left/right-arrow key press if we're not using such condition.

Comment (AndrewVebster) [reply src/ui/osx/TogglDesktop/TogglDesktopTests/TimerDescriptionFieldHandlerTests.swift]: Could not come up with a better solution. Removing this test case for now

**Review by skel35 — APPROVED** [review]
Looks good 👍

**Pull request merged**